File: code wars/6kyu/Bit Counting/main.go
package main

func CountBits(n uint) int {
	count := 0
	for n > 0 {
		count += int(n & 1)
		n >>= 1
	}
	return count
}
// real OG do it this way
import "math/bits"
var CountBits = bits.OnesCount

File: code wars/6kyu/Bit Counting/main.py
def count_bits(n):
    return bin(n).count("1") # Or return n.bit_count()

## The way I did it in go :
def countBits(n):
    total = 0
    while n > 0:
        total += n % 2
        n >>= 1
    return total
